aksel.nav.no/website/sanity/schema/objects/shared/kode.tsx

```diff
@@ -28,6 +28,8 @@ export const Kode = defineType({
           { value: "less", title: "LESS" },
           { value: "bash", title: "Terminal" },
           { value: "markdown", title: "Markdown" },
+          { value: "dockerfile", title: "Dockerfile" },
```

Comment:
https://www.npmjs.com/package/prism-react-renderer#language
`yaml` vil fungere, men må sette opp custom language-support for docker hvis man ønsker det. 

Kan hende noe slikt fungerer: https://github.com/FormidableLabs/prism-react-renderer/issues/22#issuecomment-1610204570

Reply:
OK. Da dropper vi syntax highlighting av dockerfiles for nå.